src/supertux/menu/particle_editor_open.cpp

```diff
@@ -55,11 +56,17 @@ ParticleEditorOpen::~ParticleEditorOpen()
 void
 ParticleEditorOpen::menu_action(MenuItem& item)
 {
+  std::string path;
```

Comment:
What? Can't you just make the variable inside the switch case?

Reply:
https://stackoverflow.com/questions/92396/why-cant-variables-be-declared-in-a-switch-statement

The [coding style documentation](https://github.com/SuperTux/supertux/blob/master/CODINGSTYLE.md) does not specify whether to declare variables outside the switch statement or to create a new scope within the case statement. Since I'm used to the former with SDL and I couldn't find any source as to how to format a scope within a case, I put the variable outside the block. I can move it inside and create a scope, if you could provide me with a preferred formatting template.